manager/controlapi/service_test.go

```diff
@@ -15,6 +15,20 @@ import (
 	"google.golang.org/grpc/codes"
 )
 
+func createCustomSpec(name, runtime string) *api.ServiceSpec {
+	spec := createSpec(name, runtime, 0)
+	spec.Task.Runtime = &api.TaskSpec_Custom{
+		Custom: &api.CustomSpec{
+			Name: runtime,
+			Payload: &gogotypes.Any{
+				TypeUrl: "com.docker.custom.runtime",
```

Comment:
Should we not follow the conventions laid out in the protobuf specification?

Here are some examples: https://developers.google.com/protocol-buffers/docs/reference/csharp/class/google/protobuf/well-known-types/any.

The proposed format in this PR _is not_ a url.